config/initializers/omniauth.rb

```diff
@@ -0,0 +1 @@
+OmniAuth.config.logger = Rails.logger
```

Comment:
by default it logs to STDOUT, let me know if this should continue the default behavior